pkg/sdk/capability.go

```
@@ -0,0 +1,161 @@
+
+	if rateLimit, ok := ctx.Parameters.GetFloat(RateLimitParam); ok && rateLimit > 0 {
+		cfg.RateLimit = rateLimit
+	}
+
+	results, err := brutus.BruteWithContext(context.Background(), cfg)
+	if err != nil {
+		return fmt.Errorf("brute force execution failed: %w", err)
+	}
+
+	var emitErrs []error
+	for _, r := range results {
```

golangci-lint (2.11.4)

[error] rangeValCopy: each iteration copies 168 bytes (consider pointers or indexing)

(gocritic)